pkg/controller/node/node_controller.go

```diff
@@ -65,11 +65,18 @@ const (
 	masterPoolName = "master"
 )
 
-var nodeUpdateBackoff = wait.Backoff{
-	Steps:    5,
-	Duration: 100 * time.Millisecond,
-	Jitter:   1.0,
-}
+var (
+	nodeUpdateBackoff = wait.Backoff{
+		Steps:    5,
+		Duration: 100 * time.Millisecond,
+		Jitter:   1.0,
+	}
+	// NodeUpdateTaint is a taint applied by MCC when the update of node starts.
```

Comment:
nit: when you consolidate these vars elsewhere, maybe add the "why" of this to the comment?

Reply:
Sure. Do you have a location where I can share code across MCO and MCC?

Reply:
i think we could maybe use constants.go?

https://github.com/openshift/machine-config-operator/blob/9c6c2bfd7ed498bfbc296d530d1839bd6a177b0b/pkg/constants/constants.go